tests/test_datayaml.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import pytz
+import yaml
+
+from datetime import datetime
+
+
+KEYS_CONTAINING_DATES = ['last_updated', 'expires', 'sent']
+
+
+def get_alerts_data(path):
+    with open(path) as file:
+        alerts_data = yaml.load(file, Loader=yaml.FullLoader)
+    return alerts_data
+
+
+def does_date_contain_explicit_offset(date):
```

Comment:
Minor: this could be more specific e.g. `is_date_in_london_timezone`. Similarly for the name of the test function.

Reply:
done ^_^